assets/src/blocks/amp-story-post-date/index.js

```diff
@@ -18,8 +18,8 @@ export const settings = {
 	icon: 'clock',
 	keywords: [
 		__( 'post', 'amp' ),
-		__( 'publish', 'amp' ),
-		__( 'published', 'amp' ),
+		__( 'publish date', 'amp' ),
+		__( 'published date', 'amp' ),
```

Comment:
@westonruter This is kind of redundant. Someone typing `publish` will already see the block (and probably only this block). I doubt someone will type `published` after that.

Reply:
I actually tried to enter “published” and I was surprised when the block disappeared. So I can attest to it being of some value.

Reply:
But I don't feel strongly about it.

Reply:
Alright. Let's leave it for now.